apps/tup-cms/src/taccsite_cms/settings_custom.py

```diff
@@ -205,6 +205,12 @@
 # https://github.com/django-cms/django-filer/blob/2.0.2/docs/permissions.rst
 FILER_ENABLE_PERMISSIONS = True
 
+# https://github.com/django-cms/djangocms-text-ckeditor
+CKEDITOR_SETTINGS = {
+    'autoParagraph': True, # Core-CMS had set this to False
```

Comment:
**Important.** This is what is changing. Paragraphs will automatically wrap inline elements upon editing a Text plugin instance.